rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -49,13 +82,60 @@ def test_hop_l2_deposit(database, ethereum_inquirer):
             asset=A_ETH,
             balance=Balance(amount=FVal('0.2')),
             location_label=ADDY,
-            notes='Bridge 0.2 ETH to Optimism at the same address via Hop protocol',
+            notes='Bridge 0.2 ETH to Optimism via Hop protocol',
             counterparty=CPT_HOP,
             address=string_to_evm_address('0xb8901acB165ed027E32754E0FFe830802919727f'),
+            extra_data={'sent_amount': '0.2'},
         )]
     assert expected_events == events
 
 
+@pytest.mark.vcr()
```

Comment:
can you be putting the api key as a filtered parameter for vcr. Like `@pytest.mark.vcr(filter_query_parameters=['apikey'])`

Here and elsewhere